src/OpenFeature/Telemetry/EvaluationEvent.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+
+namespace OpenFeature.Telemetry;
+
+/// <summary>
+/// Represents an evaluation event for feature flags.
+/// </summary>
+public class EvaluationEvent
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="EvaluationEvent"/> class.
+    /// </summary>
+    /// <param name="name">The name of the event.</param>
+    /// <param name="attributes">The attributes of the event.</param>
+    public EvaluationEvent(string name, Dictionary<string, object?> attributes)
+    {
+        this.Name = name;
+        this.Attributes = attributes;
+    }
+
+    /// <summary>
+    /// Gets or sets the name of the event.
+    /// </summary>
+    public string Name { get; set; }
+
+    /// <summary>
+    /// Gets or sets the attributes of the event.
+    /// </summary>
+    public Dictionary<string, object?> Attributes { get; set; }
```

Comment:
Consider making the `Name` and `Attributes` properties immutable by removing the setters or using `init`-only setters. This will ensure the `EvaluationEvent` remains a consistent snapshot of data once constructed.
Additionally, you might want to use `IReadOnlyDictionary<string, object?>` instead of `Dictionary<string, object?` for `Attributes` to reinforce the read-only intent and avoid accidental mutation.
Optionally, adding a defensive copy of the attributes in the constructor can help prevent shared reference issues:

```csharp
Attributes = new Dictionary<string, object?>(attributes);
```

Reply:
Fixed with: https://github.com/open-feature/dotnet-sdk/pull/397/commits/f91851d1adc5313b5208a8246a4b6f5f5e7d37dc